src/pages/index.js

```diff
@@ -47,12 +54,30 @@ editFormValidator.enableValidation();
 const addPicModalForm = addPicModal.querySelector(".modal__form");
 const addFormValidator = new FormValidator(validationSettings, addPicModalForm);
 addFormValidator.enableValidation();
+
+const updateAvatarFormValidator = new FormValidator(
+  validationSettings,
+  avatarModalForm
+);
+updateAvatarFormValidator.enableValidation();
+
 //
 // User Info
 //
 
 const userInfo = new UserInfo(".profile__name", ".profile__title");
 
+api
+  .getUserInfo()
+  .then((data) => {
+    console.log(data);
+    userInfo.setUserInfo(data);
+    profilePicture.src = data.avatar;
```

Comment:
`    profilePicture.src = data.avatar;
`

* Class `UserInfo` should set the entire user’s data (including the **avatar**)